pom.xml

```diff
@@ -357,6 +358,7 @@
                                 <arg>--initialize-at-build-time=org.junit.jupiter.api.parallel.ResourceLock</arg>
                                 <arg>--initialize-at-build-time=org.junit.jupiter.api.parallel.ResourceLockTarget</arg>
                                 <arg>--initialize-at-build-time=org.junit.jupiter.api.parallel.ResourceAccessMode</arg>
+                                <arg>--initialize-at-build-time=org.junit.platform.commons.support.scanning.DefaultClasspathScanner</arg>
```

Comment:
All of these initialization options seem to no longer be required on JDK 25. They're only here for 17 and 21.